config/routes.rb

```diff
@@ -1,4 +1,6 @@
 IntergalacticBattleship::Application.routes.draw do
+
+  root 'games#index'
   # The priority is based upon order of creation: first created -> highest priority.
   # See how all your routes lay out with "rake routes".
 
```

Comment:
I believe we already have a root route (a few PRs have been merged since you started, did you pull the current master back into your branch and resolve merge conflicts?)